src/__tests__/toggleselectoptions.js

```diff
@@ -0,0 +1,136 @@
+import {render, screen} from '@testing-library/react'
+import React from 'react'
+import userEvent from '..'
+import {setup, addListeners} from './helpers/utils'
+
+test('should fire the correct events for multiple select', () => {
+  const {element: select, getEventCalls} = setup(
+    <select data-testid="element" multiple>
+      <option value="1">1</option>
+      <option value="2">2</option>
+      <option value="3">3</option>
+    </select>,
+  )
+
+  userEvent.toggleSelectOptions(select, '1')
+
+  expect(getEventCalls()).toMatchInlineSnapshot(`
+    mouseover: Left (0)
+    mousemove: Left (0)
+    mousedown: Left (0)
+    focus
+    mouseup: Left (0)
+    click: Left (0)
+    mouseover: Left (0)
+    mousemove: Left (0)
+    mousedown: Left (0)
+    mouseup: Left (0)
+    click: Left (0)
+    change
+  `)
```

Comment:
Let's add an assertion here for what the final state is because currently it's unclear.

Though we *might* be able to update the `change` event in `getEventCalls` to add the before/after state. If you're up for it, then feel free to give it a try.